ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -262,6 +262,7 @@ measurement:
   angularmomentum: "The quantity of rotation of a body, which is the product of its moment of inertia and its angular velocity."
   aqi: "Air quality index, a set of special standard gradings for the pollution in air."
   area: "Amount of space within the perimeter of a 2D shape"
+  electriccharge: "Energy charge in a battery"
```

Comment:
Please revise this to be more general. For example: "The amount of energy or electrons that pass from one body to another by different modes like conduction, induction or other specific methods (e.g., the charge stored in a battery)."